legacy/messaging/src/main/java/org/jboss/as/messaging/MessagingTransformers.java

```diff
@@ -92,20 +69,14 @@ static void registerTransformers(final SubsystemRegistration subsystem) {
         buildTransformers2_1_0(chainedBuilder.createBuilder(subsystem.getSubsystemVersion(), VERSION_2_1_0));
         // 2.1.0 -> 2.0.0 (WildFly 8.0.0.Final)
         buildTransformers2_0_0(chainedBuilder.createBuilder(VERSION_2_1_0, VERSION_2_0_0));
-        // 2.0.0 -> 1.3.0 (AS7 7.3.0)
-        buildTransformers1_3_0(chainedBuilder.createBuilder(VERSION_2_0_0, VERSION_1_3_0));
-        // 1.3.0 -> 1.2.1 (AS7 7.2.1)
-        buildTransformers1_2_1(chainedBuilder.createBuilder(VERSION_1_3_0, VERSION_1_2_1));
-        // 1.2.1 -> 1.2.0 (AS7 7.2.0)
-        buildTransformers1_2_0(chainedBuilder.createBuilder(VERSION_1_2_1, VERSION_1_2_0));
-        // 1.2.0 -> 1.1.0 (AS7 7.1.3, AS7 7.1.2)
-        buildTransformers1_1_0(chainedBuilder.createBuilder(VERSION_1_2_0, VERSION_1_1_0));
+        // 2.0.0 -> 1.4.0 (AS7 7.5.0) EAP 6.4
+        buildTransformers1_4_0(chainedBuilder.createBuilder(VERSION_2_0_0, VERSION_1_4_0));
+        // 2.0.0 -> 1.3.0 (AS7 7.3.0) EAP 6.2 & EAP 6.3
```

Comment:
The comment is not consistent with the method call: the `buildTransformers1_3_0` transforms from 1.4.0 to 1.3.0, not from 2.0.0 to 1.3.0.